Arq 323

LightGuard opened the pull request:
Job Done

LightGuard committed:
Implements ARQ-323

I've tested the war portion, but I don't have a full profile Glassfish
3.1. I'm pretty confident though that it will work fine.

aslakknutsen commented:
Excellent stuff! 

We do need more info about the deployment when we upgrade to next version:
- http contexts (ip, port)
- \- servlets, name / mapping
- rmi context (ip, port)
  - remote ejbs, name / class, mapping ?
- jndi

Have you see any of these around in the Management API ?

LightGuard committed:
Fixing up the container per comments on github

Also removing the iml file.
The war test works fine incontainer, ear did not work in container,
we'll have to flesh that one out a bit.

LightGuard committed:
ignoring the iml file

aslakknutsen commented:
I'm sure you know, but just adding it to the workflow:

Rebase against arquillian/arquillian:master and update to the SPI

LightGuard committed:
ignoring other intellij files

LightGuard committed:
Merge remote-tracking branch 'upstream/master'

LightGuard committed:
Adding .idea to ignore

LightGuard committed:
Start updating to latest Arquillian API.

LightGuard committed:
Back to where we were before, it works but work still needs to be done.

We need to support username / password for admin
SSL for Admin and the rest of the server
Pulling the context root out of the web.xml
Maybe some others as well, can't remember.

aslakknutsen commented:
rebased and pushed upstream
